src/components/common/Footer/components/SocialLinks/styles.module.scss

```diff
@@ -0,0 +1,49 @@
+@import 'styles/variables';
+
+.container {
+    display: flex;
+    gap: 8px;
+}
+
+.link {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-position: center center;
+    background-repeat: no-repeat;
+}
+
+.fb {
+    background-image: url(../../../../../../public/images/fb-icon.svg);
+}
+
+.insta {
+    background-image: url(../.././../../../../public/images/insta-icon.svg);
+}
+
+.twitter {
+    background-image: url(../../.././../../../public/images/twitter-icon.svg);
+}
```

Comment:
we do not want to use this approach for icons because when you load page - browser takes some time to load it. So for a moment user will see nothing